csharp/aspnetcore/Startup.cs

```diff
@@ -2,33 +2,31 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
 
 namespace aspnetcore
 {
-public class Startup
-{
-    public void ConfigureServices(IServiceCollection services)
+    public class Startup
     {
-        services.AddRouting();
-    }
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddRouting();
```

Comment:
This isn't required

Reply:
Ah, I missed that when making the updates. Sorry!